tools/deploy_translatable_strings.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# Script for pushing the translatable strings to qiskit-translations repo.
+
+# Non-travis variables used by this script.
+SOURCE_REPOSITORY="git@github.com:Qiskit/qiskit-machine-learning.git"
+SOURCE_DOC_DIR="docs/_build/html"
+SOURCE_DIR=`$PWD/machine-learning/`
+SOURCE_LANG='en'
+
+TARGET_REPOSITORY="git@github.com:qiskit-community/qiskit-translations.git"
+TARGET_BRANCH_PO="master"
+DOC_DIR_PO="docs/locale/experiments"
```

Comment:
Is this correct?

Reply:
Fixed in https://github.com/Qiskit/qiskit-machine-learning/pull/130/commits/8e316e309ff795a1b4076aac85e903f4527c6d78